radicle/src/cob/patch.rs

```diff
@@ -530,6 +530,22 @@ pub enum State {
     Merged,
 }
 
+impl<'de> Deserialize<'de> for State {
+    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        let s = String::deserialize(deserializer)?.to_lowercase();
+        match s.as_str() {
+            "open" => Ok(State::Open),
+            "draft" => Ok(State::Draft),
+            "archived" => Ok(State::Archived),
+            "merged" => Ok(State::Merged),
+            _ => Err(serde::de::Error::custom("Unrecognised patch state")),
+        }
+    }
+}
```

Comment:
Sorry, I don't think this is any better. Now `Deserialize` isn't the inverse of `Serialize` which seems like a bad idea. The `FromStr` version is a better route or we figure out if `axum` can handle tagged enums.

Reply:
Okay I understand, will keep looking 😎